react/react-training/src/constants/index.js

```diff
@@ -6,4 +6,20 @@ export const ERROR_404 = 'Sorry. 404 page. Don`t cry.';
 
 export const OFFLINE_MESSAGE = 'Sorry. We are offline. Please come back in a few minutes.';
 
-export const OFFLINE_WE_ARE = false;
+export const WE_ARE_ONLINE = true;
```

Comment:
I think that you can change this for: `navigator.onLine` so this variable makes sense. Check this [post](https://davidwalsh.name/detecting-online)

Reply:
@henryzarza ignoro este cambio según te expliqué para qué sirve esta constante.